setup.py

```diff
@@ -158,7 +158,7 @@ def setup_package():
         scripts=['src/pymor-demo', 'distribute_setup.py', 'dependencies.py'],
         url='http://pymor.org',
         description=' ',
-        long_description=open('README.rst').read(),
+        long_description=open('README.md').read(),
```

Comment:
For this to work correctly on PYPI, one also needs 
```
long_description_content_type='text/markdown'
```
AFAIK.

Reply:
I tried reading up on that and it's a confusing mess. I assumed this wasn't needed since ```twine check``` does not complain. I'll just add it now.

Reply:
I believe for pymumps it was needed and I had to make a new release just for the formatting ...